src/stories/StyledSpacer.tsx

```diff
@@ -0,0 +1,14 @@
+export interface StyledSpacerProps {
+  showLine?: boolean;
+  spacing: number;
+}
+
+export default function StyledSpacer({ showLine, spacing }: StyledSpacerProps) {
+  let spacerClasses = '';
+
+  showLine ? (spacerClasses = ' border-t border-dfxGray-400') : null;
+
+  const margin = 4 * spacing;
+
+  return <div style={{ margin: margin + 'px 0' }} className={spacerClasses}></div>;
```

Comment:
Can use string interpolation, if it makes sense for you
```ts
const margin = `${4 * spacing}px 0`;
```

Reply:
well, i need this calculation from within different components, now it is in a sepearte function in LayoutFunctions.tsx ... so i guess i leave it like that.